unit/libcprover-cpp/verification_resultt.cpp

```diff
@@ -73,21 +63,12 @@ SCENARIO(
 
   GIVEN("A model with a failing property from C source code")
   {
-    const std::string input = R"END(
-            int main(int argc, char *argv[]) {
-                int i;
-                int arr[] = {0, 1, 2, 3, 4};
-                __CPROVER_assert(arr[i] != 3, "failure expected: arr[3] is 3");
-                return 0;
-            })END";
-
-    auto model = get_goto_model_from_c(input);
-
-    REQUIRE(model.get_goto_functions().function_map.size() == 11);
+    api_sessiont api(api_optionst::create());
+    api.load_model_from_files({"test.c"});
 
     WHEN("We run that model past the verification engine")
     {
-      const auto &results = run_verification_engine(model);
```

Comment:
If this PR isn't using the `run_verification_engine` function anymore then it should not add it.